pom.xml

```diff
@@ -120,12 +120,13 @@
                 <groupId>log4j</groupId>
                 <artifactId>log4j</artifactId>
                 <version>1.2.17</version>
+                <scope>test</scope>
```

Comment:
Note that this is the dependency*management* section. We are already declaring the scope at the places we actually use this dependency, so I think we should leave out the scope here. Actually for sf4j-log4j12 we should do the same?